src/assets/styles/styles/custom/_tables.scss

```diff
@@ -42,6 +45,10 @@
       height: 100%;
       cursor: cell;
       padding: 0;
+      overflow: hidden;
```

Comment:
Dodałam `overflow: hidden`, żeby mieć równą siatkę podczas pracy nad dropdownem, ale to na dłuższą metę nie jest dobre rozwiązanie, bo ucina "wystające" prawe fragmenty komórek. Chociaż jak dla mnie i tak tymczasowo może to sobie wyglądać lepiej niż totalnie rozjeżdżający się grafik.
![picture](https://user-images.githubusercontent.com/48785655/107128336-255ea780-68bd-11eb-83a4-b91b67091db6.png)